src/main/java/com/netgrif/application/engine/workflow/service/DataService.java

```diff
@@ -693,6 +690,12 @@ public List<Field> getImmediateFields(Task task) {
         return fields;
     }
 
+    @Override
+    public UserFieldValue makeUserFieldValue(String id) {
+        IUser user = userService.resolveById(id, true);
+        return new UserFieldValue(user.getStringId(), user.getName(), user.getSurname(), user.getEmail());
```

Comment:
I would look nicer if the `UserFieldValue` class had a constructor that would accept `IUser`

Reply:
Resolved

Reply:
why is the newly added constructor not called then as `new UserFieldValue(user)`?